### ISSUE TITLE
Double-counting fallacy on 'deaths from substance use' introduction

### ISSUE THREAD

**Issue by franasal** [owid/owid-static#4]
The [drug-use](https://ourworldindata.org/drug-use) page, states:

https://github.com/owid/owid-static/blob/a9bf8f679c6d4152d90985927d6ba5d207d035c4/drug-use.html#L5

> Collectively, smoking, alcohol and illicit drug use kills 11.8 million people each year. **This is more than the number of deaths from all cancers**.1

however, when describing the number of deaths by substance-use following the Global Burden of Disease study, the definition includes cancer as the main indirect death cause:

> Alcohol and illicit drug use, for example, increases the risk of premature death from diseases and injury, including suicide, liver disease, hepatitis, and HIV. Smoking is a key risk factor for **lung and other forms of cancer**, heart disease, stroke and diabetes.

It is clear that the first claim comes from the referenced article but it represents a double-counting fallacy in the context of how this count is presented within the OWID project.
Appealing to the transparency values of OWID I have created this issue to kindly ask for the amendment of this fallacy by reformulating the [introductory sentence](https://github.com/owid/owid-static/blob/a9bf8f679c6d4152d90985927d6ba5d207d035c4/drug-use.html#L5).

**Comment by iezqrom** [comment]
I agree that in the introduction the logic does not flow well. However, this is very well explained and broken down below in the rest of the article (direct vs indirect). I reckon that amending the introduction to reflect the results and conclusions developed further down would help tackling the issue that you mention here.

I do think there's a major terminological issue in this article though. The use of drug-related terms is inaccurate. For instance, the statement 'smoking, alcohol and illicit drug use' mixes drugs (nouns) with behaviours (smoking) and it also implies that alcohol is not a drug. This sentence would be better framed like this: 

Collectively, drug use kills 11.8 million people each year. OR Collectively, nicotine, alcohol and other drugs use kills 11.8 million people each year.

Then: 

'Alcohol and illicit drug use...' would be more accurate like: 'Alcohol and other drugs use...'

This is a common mistake and it is seen throughout the whole text. It would be easily improved with a bit of reframing.

Great job with the website.
